For_Andrey/task_3.py

```diff
@@ -0,0 +1,241 @@
+"""Storage module for class Car, CarFabric and CarsHelperCalculationClass"""
+from math import ceil
+from random import randint
+from typing import Union
+
+
+class Car:
+    """ A class to represent a car """
+
+    NEW_CAR_COST = 10000
+    REPAIR_FOR_DIESEL_CAR = 700
+    REPAIR_FOR_PETROL_CAR = 500
+    MILEAGE_BEFORE_REPAIR_DIESEL = 150_000
+    MILEAGE_BEFORE_REPAIR_PETROL = 100_000
+    DIESEL_FUEL_COST = 1.8
+    PETROL_FUEL_COST = 2.4
+    FUEL_CONSUMPTION_DIESEL_ON_HUNDRED_KM = 0.06
+    FUEL_CONSUMPTION_PETROL_ON_HUNDRED_KM = 0.08
+    DECREASE_VALUE_DIESEL_PER_THOUSAND_KM = 105
+    DECREASE_VALUE_PETROL_PER_THOUSAND_KM = 95
+    DIESEL_ENGINE_TYPE = 'diesel'
+    PETROL_ENGINE_TYPE = 'petrol'
+
+    def __init__(self, engine: str, gas_tank: int):
+        """
+        :param engine: type engine in the car
+        :param gas_tank: volume gas_tank in the car
+        """
+        assert engine in ('diesel', 'petrol'), f'Incorrect engine type: {engine}'
+        assert gas_tank in (60, 75), f'Incorrect gas_tank type: {gas_tank}'
+        self.engine = engine
+        self.gas_tank = gas_tank
+        self.mileage = randint(55_000, 286_000)
+
+    def number_of_repairs(self) -> int:
+        """
+        Gets engine type and calculate repairs
+
+        :return:number of repairs for car
+        """
+        number_of_repairs = Car.MILEAGE_BEFORE_REPAIR_DIESEL if self.engine == Car.DIESEL_ENGINE_TYPE else \
```

Comment:
incorrect variable name - number_of_repairs 
should be something like - mileage_before_repair

Reply:
fixed